api/v1beta1/machinedeployment_types.go

```diff
@@ -162,6 +162,11 @@ type MachineDeploymentStrategy struct {
 	// MachineDeploymentStrategyType = RollingUpdate.
 	// +optional
 	RollingUpdate *MachineRollingUpdateDeployment `json:"rollingUpdate,omitempty"`
+
+	// Remediation controls how scaling remediations
```

Comment:
This sentence seems a bit off

Or I just don't know what "scaling remediations" are :)

Reply:
I'll rephrase